pkgs/tools/security/psudohash/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+  stdenv,
+  python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "psudohash";
+  version = "unstable-2023-05-15";
+
+  src = fetchFromGitHub {
+    owner = "t3l3machus";
+    repo = "psudohash";
+    rev = "2d586dec8b5836546ae54b924eb59952a7ee393c";
+    hash = "sha256-l/Rp9405Wf6vh85PFrRTtTLJE7GPODowseNqEw42J18=";
+  };
+
+  nativeBuildInputs = [ python3 ];
+  dontBuild = true;
```

Comment:
This line can be dropped as there is no makefile and stdenv won't attempt a build.